src/constants/tokenSafety.tsx

```diff
@@ -3,8 +3,9 @@ import { TokenStandard } from 'graphql/data/__generated__/types-and-hooks'
 import { SearchToken } from 'graphql/data/SearchTokens'
 
 import { ZERO_ADDRESS } from './misc'
+import { COMMON_BASES } from './routing'
 import { NATIVE_CHAIN_ID } from './tokens'
-import WarningCache, { TOKEN_LIST_TYPES } from './TokenSafetyLookupTable'
+import TokenSafetyLookupTable, { TOKEN_LIST_TYPES } from './TokenSafetyLookupTable'
```

Comment:
Importing the default w/ same name as the class is confusing to me, makes it look like you're accessing static methods of the class rather calling methods of an instance of it. Exporting as non-default here probably is the better option

Reply:
The class is never actually exported, only a singleton, so the filename is misleading.

Reply:
Updated to `tokenLogoLookup`, `tokenSafetyLookup`.